cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -103,9 +103,9 @@ With `IF EXISTS`, no error is thrown and nothing happens should the index not ex
 
 | [source, cypher]
 ----
-CALL db.indexes
+SHOW [ALL\|BTREE] INDEX[ES] [BRIEF\|VERBOSE [OUTPUT]]
 ----
-| List all indexes in the database. 
+| List indexes in the database, either all or btree only.
```

Comment:
```suggestion
| List indexes in the database, either all or B-tree only.
```
We say _B-tree_ in the Operations Manual, but I realize the use is not consistent across all manuals, so this is only a suggestion.